website/src/pages/pluginhub.js

```diff
@@ -128,7 +128,7 @@ function Plugins(props) {
   const { siteConfig } = useDocusaurusContext();
   const sidebar = siteConfig.customFields.plugins.map((section) => {
     return (
-      <Sidebaritem><a className="sidebar-link" href={`#${section.groupName}`}>{section.groupName}</a></Sidebaritem>
+      <Sidebaritem key={xxx}><a className="sidebar-link" href={`#${section.groupName}`}>{section.groupName}</a></Sidebaritem>
```

Comment:
oh no, I mean there should have a unique key, see React Keys

Reply:
@juzhiyuan  I think I have already  changed 🤔
![s](https://user-images.githubusercontent.com/53715187/128621136-2196d16f-2c0f-49d1-a1f7-11990d830d09.jpg)